code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -761,10 +761,10 @@
 	user.visible_message("<span class='notice'>[user] opens \the [src] and starts pumping the handle.</span>", \
 						"<span class='notice'>You open \the [src] and start pumping the handle.</span>")
 	while(recharging)
-		if(!do_after(user, 10, src))
+		if(!do_after(user, 10, src, ignore_movement = TRUE))
```

Comment:
This has been discussed in the past, and, IIRC, it was decided that being stationary while recharging the phaser was intentional.

Reply:
Regarding playtesting and such, it _sucks_ to just have to stand still for 30+ seconds. 

I'm still in favor of keeping the ignore_movement flag.

Reply:
It was intended. Basically, it gives time for explorers to pause and talk/RP stuff out, and discourages just running through the whole expedition area without RPing at all.

Reply:
Is it still possible to disagree and request the change be kept?

Reply:
In this particular case, no, unfortunately.

Reply:
Flag removed.